getting-started/README.md

```diff
@@ -16,21 +16,21 @@ Please refer to the guide for full instructions on how to run the samples.
 
     $ cd $GOPATH/src/github.com/GoogleCloudPlatform/golang-samples/getting-started/helloworld
     $ go run helloworld.go
-    $ aedeploy gcloud preview app deploy app.yaml --set-default --docker-build=remote
+    $ aedeploy gcloud app deploy app.yaml --set-default --docker-build=remote
```

Comment:
meep.  --docker-build=remote doesn't exist anymore, it's just how it works.  Same for --set-default.  This really only needs to be:

`aedeploy gcloud app deploy`